src/bindings/python/tests/runtime.py

```diff
@@ -117,6 +117,9 @@ def __call__(self, *input_values: NumericData) -> List[NumericData]:
         input_shapes = [get_shape(input_value) for input_value in input_values]
         if self.network_cache.get(str(input_shapes)) is None:
             model = self.model
+            if self.model.is_dynamic():
+                model = model.clone()
+                model.reshape(dict(zip(param_names, [PartialShape(i) for i in input_shapes])))
```

Comment:
joke!
You could've continued it this way :D
```suggestion
                model.reshape(dict(zip(param_names, map(PartialShape, input_shapes)))
```